src/usd/sdf_usd_parser/world.cc

```diff
@@ -80,6 +82,33 @@ namespace usd
         std::cerr << "Error parsing model [" << model.Name() << "]\n";
         return false;
       }
+      bool isArticulated = false;
+      if (model.JointCount() > 0)
+      {
+        for (uint64_t j = 0; j < model.JointCount(); j++)
+        {
+          const auto joint = *(model.JointByIndex(j));
+          if (joint.Type() == sdf::JointType::REVOLUTE)
+          {
+            isArticulated = true;
+            break;
+          }
+        }
+      }
+      if (isArticulated)
```

Comment:
instead of having a separate `if` check here, what about moving the code in this `if` block to the `if (joint.Type() == sdf::JointType::REVOLUTE)` block above? That would also remove the need for the `isArticulated` flag